src/components/interfaces/MOBILE_API.xml

```diff
@@ -3100,16 +3100,21 @@
         <param name="hdRadioEnable" type="Boolean" mandatory="false"  since="5.0">
             <description> True if the hd radio is on, false if the radio is off</description>
         </param>
-        <param name="availableHDs" type="Integer" minvalue="1" maxvalue="7" mandatory="false" since="5.0">
-            <description>number of HD sub-channels if available</description>
+        <param name="availableHDs" type="Integer" minvalue="0" maxvalue="7" mandatory="false" deprecated="true" since="5.2">
```

Comment:
Also I think there was a mistake in the proposal. availableHDs should not be changed to a minvalue of 0. The parameter should only be depreacted, the overall behavior of the parameter should not be changed in the same version as well. 

I believe the mention:

> This proposal changes the minimum index of HD radio sub-channels from 1 to 0.

... should only apply to the new parameters replacing these deprecated ones.

Reply:
@JackLivio Thanks. Fixed in be2dd5badfc9fef8734b56fce1a1827f8ac1aee0